gef.py

```diff
@@ -1003,19 +1003,25 @@ def psprint(self):
         return "\n".join(msg) + "\n"
 
 
+pattern_libc_ver = re.compile(b"glibc (\d+)\.(\d+)")
```

Comment:
do `rb"glib...` since it's a regex.

Reply:
I did this in the first commit, but then it did not pass `make lint`, which executes `python2 -m py_compile gef.py`. Python2 don't understand this syntax. If I am not mistaken, in the form of `b"glibc ..."` it works correctly.
Should I still change to `rb""`?

Reply:
don't worry about Python2 because we are planning to remove support for it this year

Reply:
Yeah, we will fix the linter